include/grpc/event_engine/event_engine.h

```diff
@@ -379,6 +379,11 @@ class EventEngine : public std::enable_shared_from_this<EventEngine> {
   ///
   /// \a Closures scheduled with \a Run cannot be cancelled. The \a closure will
   /// not be deleted after it has been run, ownership remains with the caller.
+  ///
+  /// Implementations must not immediately execute the closure in a blocking
```

Comment:
"Implementations must not execute the closure in the calling thread before Run() returns"?

I think there's a few too many extra words here that make it less clear. I'd certainly strike 'in a blocking manner'.

Reply:
updated.